### ISSUE TITLE
PTN000012880 / PTN000076140 / PTN000329966

### ISSUE THREAD

**Issue by sjm41** [geneontology/go-annotation#4450]
* **PTHR ID & PTN node:**
PTN000012880 : CYTOCHROME C OXIDASE SUBUNIT 5B, MITOCHONDRIAL (PTHR10122)
PTN000076140 : CYTOCHROME C OXIDASE SUBUNIT IV (PTHR10707)
PTN000329966 : CYTOCHROME C OXIDASE SUBUNIT VIIC (PTHR13313)

These nodes add "contributes_to cytochrome-c oxidase activity" to supernumerary (ie non-catalytic) subunits of the CYTOCHROME C OXIDASE COMPLEX.
E.g. see these reviews: PMID:17091399 and PMID:28870773

Source for each one is a S. cerevisiae annotation:
PTN000012935 (YEAST|SGD=S000003155|UniProtKB=P04037)
PTN000842470 (YEAST|SGD=S000001373|UniProtKB=P00425) & PTN000842469 (YEAST|SGD=S000004997|UniProtKB=P00424)
PTN000999737 (YEAST|SGD=S000004387|UniProtKB=P04039)

Can you remove these 'contributes to' annotations from these 3 nodes please?

**Comment by marcfeuermann** [comment]
All 3 done.

**Comment by pgaudet** [comment]
Hi, 

I'm re-opening this ticket as part of the list of genes with no human PANGO annotations. 

Questions: 
1. SGD and pombase annotate the orthologs S000003155 and SPAC1296.02 to GO:0004129 cytochrome-c oxidase activity; is this correct, or should these be removed? @ValWood @srengel 
2. From looking at PMID:28870773, the core catalytic subunits seem to be COX1, COX2 and COX3. What is the function of the non-catalytic subunits? Some kind of structural constituents? It would be great to have annotations for the other subunits,

**Comment by ValWood** [comment]
I didn't realise that the catalytic core was restricted. I agree we need to annotate to something.....

**Comment by srengel** [comment]
this is correct use of contributes_to for our yeast annotations.  what is the issue here?

**Comment by sjm41** [comment]
Here's what AI has to say on the subject:
_Cytochrome c oxidase supernumerary subunits, also called accessory subunits, primarily function to regulate the activity of the core enzyme by influencing its assembly, stability, and interaction with other mitochondrial proteins, rather than directly participating in the catalytic process of electron transfer and oxygen reduction; essentially, they fine-tune the enzyme's function depending on cellular conditions._ 

That summary is partly based on:

https://pubmed.ncbi.nlm.nih.gov/26190566
(abstract includes: "Cytochrome c oxidase (COX) from mammals and birds is composed of 13 subunits. The three catalytic subunits I-III are encoded by mitochondrial DNA, the ten nuclear-coded subunits (IV, Va, Vb, VIa, VIb, VIc, VIIa, VIIb, VIIc, VIII) by nuclear DNA. The nuclear-coded subunits are essentially involved in the regulation of oxygen consumption and proton translocation by COX, since their removal or modification changes the activity"

and

https://pubmed.ncbi.nlm.nih.gov/28988874/
(Intro includes: This enzyme increased in complexity during evolution through the incorporation of additional subunits (supernumerary subunits) to the catalytic subunits I–III.....The main function of the ten supernumerary subunits is not to improve but to regulate CcO activity)

So the consensus is that the supernumerary subunits are regulators of the COX1/2/3 catalytic activity.
Whether we can find direct experimental evidence to make 'regulator activity' type annotations for each supernumerary subunit is another matter.....

**Comment by marcfeuermann** [comment]
I think we can close this one. We still have no MF. Requires literature reading to annotate (adaptor, scaffold, regulator,...) but this is not a PAINT issue.
